secondpr

```diff
@@ -0,0 +1,11 @@
+Use GIT Bash for Microsoft Windows environment to run GIT from my commandline.
+From the twindle repository page on GitHub, click the green button labeled Clone or download.
+In the “Clone with HTTPs” section, copy the URL for the repository.
+On your laptop, open your GIT bash shell.
+Type 'cd desktop' to change to to desktop directory where you want to install the cloned repository
+Next, type 'git clone (URL for the repository)'
+Once completed, it will display 'done'
+Type 'ls -a' to view all the files in the repository (Do not edit any file from the command line).
+Now go back and make some changes and save.
+Then commit on your laptop by typing 'git commit'
+Then push to your fork and then on your fork you make pull request, just like for 1st PR.
```

Comment:
I don't think this file should have been merged. It doesn't even have a file extensions. Please review all the files carefully before merging.

Regards!